src/common/datetime/format_date_time.ts

```diff
@@ -11,19 +11,14 @@ export const formatDateTime = (dateObj: Date, locale: FrontendLocaleData) =>
 
 const formatDateTimeMem = memoizeOne(
   (locale: FrontendLocaleData) =>
-    new Intl.DateTimeFormat(
-      locale.language === "en" && !useAmPm(locale)
-        ? "en-u-hc-h23"
-        : locale.language,
```

Comment:
This was done to actually fix a bug in Chrome, can you elaborate why you removed it?

Reply:
It is not removed, the h23 hour cycle is used when the 24 hour time format is wanted. Additionally for the 12 hour format the 12h hour cycle is added. I just moved the h23 a few lines down `hour12: useAmPm(locale)`.
Now the h23 and h12 is used for all languages, not only `en`.

Reply:
The issue is that that was not working, hence this less nice solution from the time. This may have been fixed by now in Chrome, but we need to check

Reply:
Ah okay, the new gallery pages can be used for that.

Reply:
The issue was that it displayed 24:12 instead of 00:12 I believe.

Reply:
**Safari Gallery Page Example:**
![Safari Gallery Page Example](https://github.com/home-assistant/frontend/assets/43651938/a48f7a80-61ab-47ca-b2be-bbb6945b3be7)

**Chrome Gallery Page Example:**
![Chrome Gallery Page Example](https://github.com/home-assistant/frontend/assets/43651938/4c0917ae-5f56-4da6-a700-90ce625a7505)

Reply:
https://bugs.chromium.org/p/chromium/issues/detail?id=1045791

Reply:
I guess the issue is not showing because you removed `hour12`